src/verifiers/openAttestationDnsTxt.v2.test.ts

```diff
@@ -148,12 +305,66 @@ describe("OpenAttestationDnsTxt v2 document", () => {
         }
       };
       expect(
-        openAttestationDnsTxt.test(document, {
+        await verify(document, {
           network: "ropsten"
         })
-      ).toStrictEqual(true);
+      ).toStrictEqual([
+        {
+          type: "ISSUER_IDENTITY",
+          name: "OpenAttestationDnsTxt",
+          data: [
+            {
+              status: "INVALID",
+              value: "0xabcd"
+            },
+            {
+              dns: "example.tradetrust.io",
+              status: "VALID",
+              value: "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe"
+            }
+          ],
+          message: "Certificate issuer identity for 0xabcd is invalid",
+          status: "INVALID"
+        }
+      ]);
     });
-    it("should return false if no issuer has a tokenRegistry or documentStore", () => {
+    it("should return a valid fragment when document has one issuer with token registry/valid identity and a second issuer without identity", async () => {
+      const document = {
+        ...documentRopstenValidWithToken,
+        data: {
+          ...documentRopstenValidWithToken.data,
+          issuers: [
+            documentRopstenValidWithToken.data.issuers[0],
+            {
+              ...documentRopstenValidWithToken.data.issuers[0],
+              identityProof: undefined
+            }
+          ]
+        }
+      };
+
+      const fragment = await verify(document, {
+        network: "ropsten"
+      });
+      expect(fragment).toStrictEqual([
+        {
+          type: "ISSUER_IDENTITY",
+          name: "OpenAttestationDnsTxt",
+          data: [
+            {
+              dns: "example.tradetrust.io",
+              status: "VALID",
+              value: "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe"
+            },
+            {
+              status: "SKIPPED"
+            }
+          ],
+          status: "VALID"
+        }
+      ]);
+    });
+    it("should return skipped fragment if no issuer has a tokenRegistry or documentStore", async () => {
```

Comment:
i assume there's somewhere that fails when the document doesn't have a valid issuer check?

Reply:
1. https://github.com/Open-Attestation/oa-verify/blob/fix/dns-verifier/src/verifiers/openAttestationDnsTxt.v2.test.ts#L63
2. https://github.com/Open-Attestation/oa-verify/blob/fix/dns-verifier/src/verifiers/openAttestationDnsTxt.v2.test.ts#L282